RELEASE/scripts/autoscend/quests/level_08.ash

```diff
@@ -411,41 +383,39 @@ void itznotyerzitzMineChoiceHandler(int choice) {
 	}
 }
 
-boolean L8_trapperGround()
-{
-	if (internalQuestStatus("questL08Trapper") < 0 || internalQuestStatus("questL08Trapper") > 1)
-	{
-		return false;
-	}
-	if (L8_getGoatCheese() || L8_getMineOres() || L8_trapperAdvance())
-	{
-		return true;
-	}
-	return false;
-}
-
 boolean L8_trapperExtreme()
 {
-	if(get_property("currentExtremity").to_int() >= 3)
+	if(internalQuestStatus("questL08Trapper") != 2)
 	{
 		return false;
 	}
-	if (internalQuestStatus("questL08Trapper") != 2)
+	if(get_property("_sourceTerminalDigitizeMonster") == $monster[Ninja Snowman Assassin])
 	{
 		return false;
 	}
-	if(get_property("_sourceTerminalDigitizeMonster") == $monster[Ninja Snowman Assassin])
+	if(L8_trapperPeak())
 	{
-		return false;
+		return true;		//unlock peak if ready to do so.
 	}
-
-	if (possessOutfit("eXtreme Cold-Weather Gear", true)) {
+	
+	//we should equip the extreme outfit if we have it
+	if (possessOutfit("eXtreme Cold-Weather Gear", true))	//own and can equip
+	{
 		autoOutfit("eXtreme Cold-Weather Gear");
-	} else if (possessOutfit("eXtreme Cold-Weather Gear")) {
+	}
+	else if (possessOutfit("eXtreme Cold-Weather Gear"))	//just own. thanks to else can not equip
+	{
 		auto_log_warning("I can not wear the eXtreme Gear, I'm just not awesome enough :(", "red");
 		return false;
 	}
-
+	else
+	{
+		handleFamiliar("item");			//we want item drop while trying to assemble the outfit
+	}
+	
+	providePlusNonCombat(25);		//we want -combat in this zone
```

Comment:
already handled in `acquireCombatMods()`/`zone_combatMod()`

Reply:
thank you. fixed